tests/01_03_52-disable_lock_whitelist.js

```diff
@@ -1,121 +1,78 @@
-    const INITIAL_RSK_BALANCE = bitcoin.btcToSatoshis(10);
-
-    const addresses = await pegClient.generateNewAddress('test');
-    expect(addresses.inRSK).to.be.true;
+    const btcAddressInfo = await btcTxHelper.generateBtcAddress('legacy');
+    await btcTxHelper.fundAddress(btcAddressInfo.address, 2 * MINIMUM_PEGIN_VALUE_IN_BTC + btcTxHelper.getFee());
+    const btcAddressInfoBalanceInitial = Number(await btcTxHelper.getAddressBalance(btcAddressInfo.address));
+    expect(btcAddressInfoBalanceInitial).to.be.equal(2 * MINIMUM_PEGIN_VALUE_IN_BTC + btcTxHelper.getFee());
 
-    await btcClient.sendToAddress(addresses.btc, INITIAL_BTC_BALANCE);
-    await btcClient.generate(1);
-    await test.assertBitcoinBalance(addresses.btc, INITIAL_BTC_BALANCE, "Wrong initial BTC balance");
-    await wait(1000);
+    const federationAddressBalanceInitial = Number(await btcTxHelper.getAddressBalance(federationAddress));
 
     // address is not whitelisted
-    await test.assertLock(addresses, [{ address: federationAddress, amount: INITIAL_RSK_BALANCE }], { fails: true });
-    // wait for the btc to come back so we can use the assertLock method again
-    await utils.waitForBtcToReturn(addresses.btc);
-
-    const addr = await rskClient.eth.personal.importRawKey(WHITELIST_CHANGE_PK, '');
-    expect(addr.slice(2)).to.equal(WHITELIST_CHANGE_ADDR);
-    await rskClient.eth.personal.unlockAccount(addr, '');
-    // can disable the whitelist
-    await utils.sendTxWithCheck(
-      rskClient.rsk.bridge.methods.setLockWhitelistDisableBlockDelay(200),
-      (disableResult) => expect(Number(disableResult)).to.equal(1),
-      WHITELIST_CHANGE_ADDR)();
-
-    // disable whitelist doesn't work the second time
-    await utils.sendTxWithCheck(
-      rskClient.rsk.bridge.methods.setLockWhitelistDisableBlockDelay(10),
-      (disableResult) => expect(Number(disableResult)).to.equal(-1),
-      WHITELIST_CHANGE_ADDR)();
-
-    await btcClient.generate(100);
-    await wait(500);
-    await rskClient.fed.updateBridge();
-    await rskUtils.mineAndSync(rskTxHelpers);
-    await wait(500);
-
-    // address is still not able to send btc to bridge after 100 blocks
-    await test.assertLock(addresses, [{ address: federationAddress, amount: INITIAL_RSK_BALANCE }], { fails: true });
-    await utils.waitForBtcToReturn(addresses.btc);
-
-    await btcClient.generate(100);
-    await wait(500);
-    await rskClient.fed.updateBridge();
-    await rskUtils.mineAndSync(rskTxHelpers);
-    await wait(500);
-
-    // after 200 blocks the whitelist period has ended and we can send money to the bridge
-    await test.assertLock(addresses, [{ address: federationAddress, amount: INITIAL_RSK_BALANCE }]);
+    await sendPegin(rskTxHelper, btcTxHelper, btcAddressInfo, MINIMUM_PEGIN_VALUE_IN_BTC);
+    // wait for the btc to come back so we can use the sendPegin method again
+    await rskUtils.triggerRelease(rskTxHelpers, btcTxHelper);
```

Comment:
I would add some assertions here.
- After sendPegin call, btc balance reduces by MINIMUM_PEGIN_VALUE_IN_BTC and federation balance increases by the same amount
- After triggerRelease call the inverse, that way we check that the funds went and came back

Reply:
hmm , to me it seems like too many assertions which are not directly related to this TC. but ok... :)